app/Fission/Internal/Development.hs

```diff
@@ -168,5 +178,11 @@ mkConfig' = do
   let cfg = mkConfig dbPool processCtx httpManager logFunc
   return (cfg, close)
 
-pgConnectInfo :: PGConnectInfo
-pgConnectInfo = PGConnectInfo "localhost" 5432 "webapi" Nothing Nothing Nothing
+connectionInfo :: ConnectionInfo
+connectionInfo = ConnectionInfo
+  { pgDatabase = "web_api"
```

Comment:
Not sure when that got out of sync with what's in my `env.yaml`.

_Please let me know if this is wrong for some reason_

Reply:
ACTUALLY, this function should probably read from the `env.yaml`